app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -110,3 +112,22 @@ export const MenuBarBlocks = {
     },
   ],
 };
+
+export const NOTES_FORM_VALIDATION_SCHEMA = yup.object().shape({
+  title: yup.string().required("Title is required"),
+  description: yup.string().required("Description is required"),
+  contact: yup
+    .object({
+      label: yup.string().required(),
+      value: yup.string().required(),
+    })
+    .nullable()
+    .required("Assigned Contact is required."),
+  tag: yup
+    .object({
+      label: yup.string().required(),
+      value: yup.string().required(),
+    })
+    .nullable()
+    .required("Tag is required."),
+});
```

Comment:
So can't we assign multiple tags to a note? If not, what's the business reason for the same?